### ISSUE TITLE
Add picker to QR-Code generator

### ISSUE THREAD

**Issue by Ein-Tim** [corona-warn-app/cwa-website#1143]
## Feature description

Add a picker to the https://www.coronawarn.app/en/eventregistration/ page.
The fields: 
- "Typical length of stay", - **IMPLEMENTED**
- "Start time" &
- "End time"

could be enhanced by adding a picker to them, which makes it easier to enter the information.

## Problem and motivation
The fields only accept specific formatted inputs (see https://github.com/corona-warn-app/cwa-website/pull/1132#issuecomment-825760191). This can be confusing for the user. With a picker it is ensured that the right format is entered. Also the Corona-Warn-App Android and iOS app use pickers, so the user already knows them.

**Comment by bastianjoel** [comment]
I would like to implement this. 

Are there any preferences on which picker to use?

I would suggest Flatpickr (https://flatpickr.js.org/)

Another option (which I would prefer, because I think this is mainly used on desktops) is Cleave.js (https://nosir.github.io/cleave.js/) which forces the user to follow a specific pattern and adds separators automatically.

**Comment by dsarkar** [comment]
@thomasaugsten See comment above, please!

**Comment by thomasaugsten** [comment]
No preferences. This is up to you.

**Comment by Ein-Tim** [comment]
I'd like to reopen this since only the field "Typical length of stay (hours)" got a picker with #1200, but I also asked for pickers for "Start time" & "End time". 

Please reopen (:

**Comment by bastianjoel** [comment]
This is also an reply to the question from @svengabr asked in #1200:

I'm not a UI/UX expert, but in my opinion date pickers would just slow down users here. Date pickers are helpful when users are restricted to certain dates/days or if users need to look up the date via weekdays etc, but neither of this is the case for the form.  

For that reason I just added cleave.js, which forces the user to stay within the given date pattern and therefore still addresses the problem mentioned in the issue which was to ensure that the right format was entered. Another reason why I decided against a date picker is that it is hard to find a good one. 

Still I would like to hear another opinion on that. Maybe also someone knows a date picker library which would really fit into the form. With flatpickr I wasn't content because some of the open issues on the project concerned me.
